src/_js/main.mjs

```diff
@@ -53,7 +78,11 @@ ga.q = [];
 ga('create', UA_ID, 'auto');
 ga('set', 'referrer', document.referrer.split('?')[0]);
 ga('send', 'pageview');
-const firstScript = document.scripts[0];
-const scriptElement = document.createElement('script');
-scriptElement.src = 'https://www.google-analytics.com/analytics.js';
-firstScript.parentNode.insertBefore(scriptElement, firstScript);
+insertScript('https://www.google-analytics.com/analytics.js');
+
+// Dynamically either insert the dark or the light themed Twitter widget.
+const twitterTimeline = document.querySelector('.twitter-timeline');
+if (twitterTimeline) {
+  twitterTimeline.dataset.theme = darkModeToggle.mode;
+  insertScript('https://platform.twitter.com/widgets.js');
```

Comment:
We’ll want to add `insertScript` + `insertScript('https://platform.twitter.com/widgets.js');` to `legacy.js`  as well.